IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionCellVM.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SSNewsCollectionCellVM.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class SSNewsCollectionCellVM {
```

Comment:
Adding prefixes to classes was common on Obj-c (because there were no namespaces), but I think we should avoid it on Swift. It's part of our guidelines [here](https://github.com/MakingSense/code-style-guides/blob/master/Swift/Readme.md#class-prefixes).

The MS prefix (e.g. `MSButton`) could be an exception because we want to make it very obvious it's part of our 'framework/reusable classes'.

So, unless there's a compelling reason, I prefer we avoid prefixes on Swift.